src/components/ThemeProvider/ThemeProvider.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import ThemeProvider from './ThemeProvider';
+import Button from '../Button';
+import Test from './Test';
+
+export default {
+  title: 'Original Components/ThemeProvider',
+  component: ThemeProvider,
+};
+
+export const SettingUpAButtonExample = {
+  render: () => (
+    <ThemeProvider
+      theme={{
+        palette: {
+          primary: '#2D2D46',
+        },
+      }}
+    >
+      <Button>Hello world</Button>
+    </ThemeProvider>
+  ),
+
+  name: 'Setting up a button example',
```

Comment:
story should be named "Theme with button example" or "Setting up a button example" to be consistent with the sidebar

Reply:
fixed here https://github.com/Orfium/orfium-ictinus/pull/726/commits/7820e6500d835ff3b623a3adc9ae33cf85e1fb64